plugins/azure-devops/README.md

```diff
@@ -70,7 +70,7 @@ dev.azure.com/project: <project-name>
 dev.azure.com/build-definition: <build-definition-name>
 ```
 
-In this case `<project-name>` will be the name of your Team Project and `<build-definition-name>` will be the name of the Build Definition you would like to see Builds for. If the Build Definition name has spaces in it make sure to put quotes around it
+In this case `<project-name>` will be the name of your Team Project and `<build-definition-name>` will be the name of the Build Definition you would like to see Builds for, and it's possible to add more Builds separated by a comma. If the Build Definition name has spaces in it make sure to put quotes around it.
```

Comment:
This should be "Build Definitions" just to keep this clear. Ideally we would have called this out with an example as well